assets/javascripts/discourse/controllers/admin-plugins-explorer.js.es6

```diff
@@ -33,6 +34,7 @@ export default Ember.Controller.extend({
     const id = parseInt(this.get('selectedQueryId'));
     const item = this.get('content').find(q => q.get('id') === id);
     !isNaN(id) ? this.set('showRecentQueries', false) : this.set('showRecentQueries', true);
+    if (id<0) this.set('editDisabled', true);
```

Comment:
Did you run the linter on this file? Seems like this wouldn't pass.

Reply:
Prettier wants to obliterate this file :sweat_smile: Can I fix all the linting/prettier issues in a separate PR?

(I did fix that line though^)